tests/gtest/test_fims_math_logit.cpp

```diff
@@ -0,0 +1,44 @@
+#include "gtest/gtest.h"
+#include "common/fims_math.hpp"
+
+namespace
+{
+
+  // Test logit using multiple input values
+  TEST(logit, use_multiple_input_values)
+  {
+    std::vector<double> max_value = {1.0, 10.0, 20.5};
+    std::vector<double> min_value = {0.0, 0.2, 0.05};
+    std::vector<double> x_value = {0.5, 2.0, 4.5};
+
+    // R code that generates true values for the test
+    // -log(max_value - x_value) + log(x_value - min_value)
+    // =  0.000000 -1.491655 -1.279685
+    std::vector<double> expect_value = {0.0, -1.491655, -1.279685};
+
+    for (int i = 0; i < expect_value.size(); ++i)
+    {
+      EXPECT_NEAR(fims::logit(min_value[i], max_value[i], x_value[i]), 
+      expect_value[i], 0.0001);
+    }
+  }
+
+  TEST(inv_logit, use_multiple_input_values)
+  {
+    std::vector<double> max_value = {1.0, 10.0, 20.5};
+    std::vector<double> min_value = {0.0, 0.2, 0.05};
+    std::vector<double> logit_x_value = {0, -1.491655, -1.279685};
+
+    // R code that generates true values for the test
+    // max_value - (max_value - min_value)/(1+exp(-logit_x_value))
+    // 
+    std::vector<double> expect_value = {0.5, 2.0, 4.5};
+
+    for (int i = 0; i < expect_value.size(); ++i)
+    {
+      EXPECT_NEAR(fims::inv_logit(min_value[i], max_value[i], logit_x_value[i]), 
+      expect_value[i], 0.0001);
+    }
+  }
+
```

Comment:
Add an additional test to see if the logit input to the inv_logit returns the same value and vice versa,
EXPECT_EQ(inv_logit(0,1,logit(0,1,0.5), logit(0,1,0.5))
This type of test can also be useful for testing the bounds.